console/src/main/webapp/manager/app/components/delegations/delegations.tpl.html

```diff
@@ -0,0 +1,35 @@
+<section class="delegations" ng-if="profile === 'SUPERUSER'">
+
+  <div class="col-md-12">
+
+    <input type="text" class="filter-table form-control pull-right" ng-model="delegations.q" placeholder="{{'delegations.filter' | translate}}"></input>
+    <i class="glyphicon glyphicon-remove-sign filter-table-reset" ng-show="delegations.q != ''" ng-click="delegations.q = ''"></i>
+    <h3 class="delegations-title">{{delegations.delegations.length}} <span translate>nav.delegations</span></h3>
+
+    <table class="table table-striped table-condensed">
+
+      <thead>
+        <tr>
+          <th translate>home.user</th>
+          <th translate>nav.orgs</th>
+          <th translate>nav.roles</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <tr dir-paginate="delegation in delegations.delegations | filter:delegations.q | itemsPerPage: delegations.itemsPerPage">
+          <td>
+            <a ng-link="user({id: delegation.uid, tab: 'delegations'})">{{::delegation.uid}}</a>
+          </td>
+          <td>{{::delegation.orgs.join(', ')}}</td>
```

Comment:
Can you truncate orgs/roles list if it's too long and add a complete list when mouse is over (title/alt) ?

Reply:
Using CSS with eg `text-overflow: ellipsis;` ?

Reply:
We will do this in another PR